app/view_models/prior_authority/v1/decision.rb

```diff
@@ -0,0 +1,14 @@
+module PriorAuthority
+  module V1
+    class Decision < BaseViewModel
+      attribute :laa_reference, :string
+      attribute :submission
+
+      delegate :state, to: :submission
+
+      def comments
+        submission.events.find_by(event_type: 'Event::Decision').details['comment']
```

Comment:
this will (potentailly) be wrong when appeals come in... can I suggest:
```
submission.events.order(created_at: :desc).find_by(event_type: 'Event::Decision').details['comment']
```
might be worth investing in scopes at this point. :)